src/Discovery/Discovery.tsx

```diff
@@ -89,7 +89,7 @@ export const renderFieldContent = (content: any, contentType: 'string'|'paragrap
     if (Array.isArray(content)) {
       return content.join(', ');
     }
-    return content.toLocaleString();
+    return content ? content.toLocaleString() : 'N/A';
```

Comment:
Why not use valueIfNotAvailable to handle missing values? Just like what we were doing in existing codes

Reply:
That works, however, it requires the configuration to ensure all fields are using `valueIfNotAvailable`
But that is cleaner solution